test/Microsoft.AspNetCore.Server.KestrelTests/ChunkedRequestTests.cs

```diff
@@ -77,7 +77,7 @@ public static TheoryData<ServiceContext> ConnectionFilterData
                         "0",
                          "");
                     await connection.ReceiveEnd(
-                        "HTTP/1.0 200 OK",
+                        "HTTP/1.1 200 OK",
```

Comment:
This is an odd test (expand above to see all) because HTTP/1.0 clients would hardly support chunked encoding. In any case, no reason to reject such a request. Plus, we're sending the right response. RFC 7230 section 3.3.1 says:

> A server MUST NOT send a response containing Transfer-Encoding unless the corresponding request indicates HTTP/1.1 (or later).

We implement this exact behavior in Frame.cs. You can see in this PR I've added a comment remarking that should never change.